app/components/primer/beta/text.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Primer
+  module Beta
+    # `Text` is a wrapper component that will apply typography styles to the text inside.
+    class Text < Primer::Component
+      status :beta
```

Comment:
I assume we'd eventually get rid of the `status` macro, right?

Reply:
yeah :)